cdap-cli/src/main/java/co/cask/cdap/cli/command/CallServiceCommand.java

```diff
@@ -114,9 +127,10 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
 
   @Override
   public String getPattern() {
-    return String.format("call service <%s> <%s> <%s> [headers <%s>] [body <%s>]",
+    return String.format("call service <%s> <%s> <%s> [headers <%s>] [body <%s>] [body:file <%s>]",
```

Comment:
CLI will auto-complete to either, but if you send a `body` and `body:file`, then the command will throw an exception. This isn't very good since nothing in the pattern indicates this, but I can add a note to the description.